lib/python/qmk/commands.py

```diff
@@ -28,11 +28,16 @@ def create_make_command(keyboard, keymap, target=None):
         A command that can be run to make the specified keyboard and keymap
     """
     make_args = [keyboard, keymap]
+    make_cmd = 'make'
+
+    platform_id = platform.platform().lower()
+    if 'freebsd' in platform_id:
+        make_cmd = 'gmake'
```

Comment:
I prefer avoiding OS checks whenever possible. Some quick spot checks show that this should DTRT on linux and macos too:

```suggestion
    make_cmd = 'gmake' if shlex.which('gmake') else 'make'
```

Reply:
@skullydazed I can't seem to find `shlex.which` documentation, nor does it seem to work in FreeBSD if I try your suggested change.

That being said, I get the sentiment on not wanting the OS check, let me try to see how I can check the path for a working `gmake` and only use it it found.

Reply:
Did you mean `shutil.which`?

Reply:
Yes, sorry, it should be shutil. I tested in a window and then retyped here rather than copy and paste. Sorry for the confusion!

Reply:
No worries. I just pushed the change, but using `shutil.which`.

Tested the branch in my Fedora environment, and it does pick up `gmake` there and run it, but `gmake` is just a symlink to `make` there anyways.

Reply:
It figures fedora would be the distro to do that. :)